templates/download/server/manual.html

```diff
@@ -1,213 +1,382 @@
 {% extends "download/_base_download.html" %}
 
-{% block title %}Get Ubuntu Server | Download{% endblock %}
+{% block title %}Get Ubuntu Server{% endblock %}
+
 {% block meta_description %}Get Ubuntu Server one of three ways; by using Multipass on your desktop, using MAAS to provision machines in your data centre or installing it directly on a server.{% endblock meta_description %}
+
 {% block meta_copydoc %}https://docs.google.com/document/d/1Qob4Ni_y1ffIdSWqaw2xH0dxiHZQmkRGP5nuoBhIfyM/edit{% endblock meta_copydoc %}
 
+{% block body_class %}is-paper{% endblock body_class %}
+
 {% block content %}
 
-<section class="p-strip--suru-topped">
-  <div class="row u-equal-height">
-    <div class="col-7">
-      <h1>Get Ubuntu Server</h1>
-      <h2 class="js-download-option">
-        Option 1: Manual server installation
-      </h2>
-      <p>
-        USB or DVD image based physical install
-      </p>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">OS security guaranteed until April 2027</li>
-        <li class="p-list__item is-ticked">Expanded security maintenance until April 2032</li>
-        <li class="p-list__item is-ticked">Commercial support for enterprise customers</li>
-      </ul>
-    </div>
-    <div class="col-4 col-start-large-8 u-hide--medium u-hide--small u-align--center u-vertically-center">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/b743f1ee-jj-in-circle.png",
-        alt="",
-        width="250",
-        height="250",
-        hi_def=True,
-        loading="auto"
-        ) | safe
-      }}
-    </div>
+<section class="p-strip is-shallow u-no-margin--bottom">
```

Comment:
Can you please change `u-no-margin--bottom` with `u-no-padding--bottom` here? It will address the excessive spacing.